### DISCUSSION TITLE
Exporting different association types separated by columns

### DISCUSSION THREAD

**Discussion by aharonach** [atrocore/atrocore#667] [Q&A]
Hi,
Is there a way to configure an export that exports different associations into separate columns?
In the export config screen, I can only choose the "Associated Product: Related" field, and all the associations are exported in a single column.
For example, if I have 2 different associations of "cross-sell" and "up-sell" to a product, how can I export them separately?
I couldn't find any reference to this in your documentation as well.

Thank you!

**Comment by atrolex** [comment]
Yes, this is possible, for this you need the module Connector, which gives you the possibility to apply filters to relations, just like for the main entity. So you need to create 2 columns for same relation (on Associated Product) and apply different filters to these rules.
